translations/README.russian.md

```diff
@@ -0,0 +1,199 @@
+
+## Demos
+
+Ознакомьтесь со всеми примерами приложений в 
+[community/examples](../community/sample-apps) папке.
+
+### Приложения в режиме реального времени
+
+- Приложение группового чата, построенное с помощью React, включает в себя индикатор набора текста, онлайн пользователей и уведомления о сообщениях.
+  - [Попробуйте это](https://realtime-chat.demo.hasura.app/)
+  - [Руководство](../community/sample-apps/realtime-chat)
+  - [Просмотреть API](https://realtime-chat.demo.hasura.app/console)
+
+- Приложение для отслеживания местоположения в реальном времени, показывающее движущееся транспортное средство, изменяющее текущие GPS-координаты, движущееся по карте.
+  - [Попробуйте это](https://realtime-location-tracking.demo.hasura.app/)
+  - [Руководство](../community/sample-apps/realtime-location-tracking)
+  - [Просмотреть API](https://realtime-location-tracking.demo.hasura.app/console)
+
+- Информационная панель в режиме реального времени для агрегирования данных по постоянно меняющимся данным.
+  - [Попробуйте это](https://realtime-poll.demo.hasura.app/)
+  - [Руководство](../community/sample-apps/realtime-poll)
+  - [Просмотреть API](https://realtime-poll.demo.hasura.app/console)
+
+### Видео
+
+* [Добавьте GraphQL к экземпляру GitLab, размещенному на вашем сервере](https://www.youtube.com/watch?v=a2AhxKqd82Q) (*3:44 мин*)
+* [Приложение Todo с Auth0 и GraphQL бэкенд](https://www.youtube.com/watch?v=15ITBYnccgc) (*4:00 мин*)
+* [GraphQL на GitLab интегрированный с GitLab auth](https://www.youtube.com/watch?v=m1ChRhRLq7o) (*4:05 мин*)
+* [Таблица для 10 миллионов ездок с геолокацией (PostGIS, Timescale)](https://www.youtube.com/watch?v=tsY573yyGWA) (*3:06 мин*)
+
+
+## Поддержка и устранение неисправностей
+
+Документация и сообщество поможет вам решить большинство проблем. Если вы столкнулись с ошибкой или вам нужно связаться с нами, вы можете использовать один из следующих каналов связи:
+
+* Поддержка и обратная связь: [Discord](https://discord.gg/vBPpJkS)
+* Проблема и отслеживание ошибок: [GitHub issues](https://github.com/hasura/graphql-engine/issues)
+* Следите за обновлениями продукта: [@HasuraHQ](https://twitter.com/hasurahq)
+* Поговорите с нами [чат на сайте](https://hasura.io)
+
+Мы стремимся создать открытую и доброжелательную атмосферу в сообществе. Пожалуйста, ознакомьтесь с [Кодекс поведения](../code-of-conduct.md).
+
+Если вы хотите сообщить о проблеме безопасности, пожалуйста, [прочитайте это](../SECURITY.md).
+
+## Контрибуция
+
+Ознакомьтесь с нашим [руководством по контрибуции](../CONTRIBUTING.md) для более подробной информации.
+
+## Элементы бренда
+
+Элементы бренда Hasura (логотипы, Hasura значки и т.д.) можно найти в [assets/brand](../assets/brand) папке.  Можете использовать их в вашем приложении / веб-сайте и т.д. 
+Мы были бы рады, если бы вы добавили "Powered by Hasura" бейдж к вашим приложениям, построенным с помощью Hasura.. ❤️
+
+<div style="display: flex;">

```

Comment:
The Hasura logo doesn't show properly here:

![image](https://user-images.githubusercontent.com/5722022/68928410-03f13780-07b0-11ea-91d3-89abc35f6a71.png)

Reply:
Done finally (I hope:D)